content/swiftui/concepts/viewmodifier/terms/border/border.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.border()'
+Description: 'Applies a border to a view, which can be customized with a specified style and width.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.border()`** view modifier in SwiftUI can be used to add a border to a view. The border can be customized with a specified style and width.
+
+## Syntax
+
+```pseudo
+View
+    .border(_ content: ShapeStyle, width: CGFloat = 1)
```

Comment:
For the parameters, can we change `_ content: ShapeStyle` to just `content`

Reply:
In addition, can we take away the `= 1` for the second parameter as this is included in the explanation below

Reply:
Thank you for the suggestion, I have added those changes.